docs/getting-started.md

```diff
@@ -147,7 +147,7 @@ While you can use any editor of your choice to develop your app, you will need t
 
 We recommend installing Node, Watchman, and JDK using [Homebrew](http://brew.sh/). Run the following commands in a Terminal after installing Homebrew:
 
-```
+```sh
 brew install yarn
 brew install node
 brew install watchman
```

Comment:
Btw, slightly off-topic, are the below Java related dependencies necessary for iOS projects, or can that be scoped to the Android setup instructions?

Reply:
yep, it struck me as well, will update

Reply:
I've split JDK from Homebrew and Node to achieve that